plutok-flat/src/main/kotlin/io/newm/plutok/uplc/ast/Constant.kt

```diff
@@ -41,3 +49,27 @@ data class ApplyConstant(val value: Constant, val type: Type) : Constant
 
 // tag: 8
 data class DataConstant(val value: PlutusData) : Constant
+
+object ConstantSerializer : KSerializer<Constant> {
+    override val descriptor = buildClassSerialDescriptor("Constant")
+
+    private const val CONSTANT_TAG_WIDTH = 4
+
+    override fun deserialize(decoder: Decoder): Constant {
+        TODO()
```

Comment:
Is this TODO later or not needed at all?

Reply:
Later. I'm only building out the serializing to Flat files right now. Deserializing FROM flat files will come later.